EternalBAND.Win/Controllers/WebApi/ErrorWebController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Azure;
+using EternalBAND.Api.Helpers;
+using EternalBAND.Api.Services;
+using EternalBAND.DomainObjects.ViewModel;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+
+namespace EternalBAND.Win.Controllers.WebApi
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ErrorWebController : ControllerBase
+    {
+        public ErrorWebController()
+        {
+        }
+
+        [HttpGet, Route("webError/{code}")]
+        public IActionResult Error(int code)
+        {
+            switch (code)
+            {
+                case (int)HttpStatusCode.NotFound :
+                    return NotFound("page is not found");
+                case (int)HttpStatusCode.Unauthorized:
+                    return Unauthorized("USer is not logged in for this action");
```

Comment:
USer

United States with er <3

Reply:
no no in german -er postfix means .. from for ex Hamburger : from Hamburg
USer means from us : bizdensin dostum :D

Reply:
OMFG

Reply:
controller is deleted and middleware is added